### ISSUE TITLE
2024_feb_roadmap Bug: Data UI  expected clinics not showing in clinic selection list

### ISSUE THREAD

**Issue by dlkibbe** [lzim/teampsd#3054]
**1. Add description:** 

[Site name redacted] - The site has been "re-vamping" their clinic assignment process and changes in MH staffing over the past 2 years have been significant. Participant noted that there are clinics missing in her clinic selection options. Specifically, there are "many, many [Site name redacted]clinics" and most are not showing up in the data UI clinic selection list.

<< Paste screenshots here>>
1. When was the bug discovered? Date: 2024_01_11  Time: 2:00 to 2:45 pm
2. Is this an invidivual or shared world? (Do not name specific world, keep de-identified) neither data UI
3. What is the name of the experiment? (Ok to name, not identifiable). not applicable

**2. In order for Team PSD to track this issue you must assign
- [ X] issues/bugs -  Select **bug_tracker** under project, on the right to initiate tracking a Bug.
- [ ] features 
- [ ] manuscript 

**3. Support Workflow:**  
i. **Pipelines** - prioritize in priority pipeline  
ii. **Assignee** - assign to a workflow user  
iii. **Label**- label value stream and workflow 
iv. **Milestones & Epics** - select monthly sprint milestone (GitHub) & epic (ZenHub) within the workgroup   
v. **Estimate** - estimate task in complexity points   
vi. **Epics** - assign to team-wide epic and TeamPSD master plan project  

**3. If you anticipate that your **issue card** or **bug card** will have **dependencies** these can be added once the issue has been submitted.   
viii=. **Add dependencies** - assess for and add dependencies *blocking* & *blocked by* this issue

**Comment by lijenn** [comment]
Reminder to please remove clinic names as we don't want them on our public facing repo! I've redacted them in this card. Thanks!

**Comment by lzim** [comment]
Close this stale card.
Clinic Selections for Mental Health Series Stop codes 500 have been updating daily.

**Comment by hnorthcraft** [comment]
@lzim @lijenn is there another card to cross ref with the res comment above that the 500 stop codes are updated dail? #adjudicate_memo

**Comment by lijenn** [comment]
@hnorthcraft No, the MH 500 stop codes are updated daily as a part of how we built our daily refreshes. This issue may be related to a mix of the daily refreshes being paused for some reason (e.g., no one accessed the VISN report for some time) and/or user error.

**Comment by hnorthcraft** [comment]
thanks @lijenn I will go ahead and mark as 0
